### DISCUSSION TITLE
Предоставление статических файлов в Express ⚡️ Node.js с примерами кода

### DISCUSSION THREAD

**Discussion by giscus** [xsltdev/nodejsdev.ru#51] [Announcements]
# Предоставление статических файлов в Express ⚡️ Node.js с примерами кода

Справочник Node.js с примерами кода

https://nodejsdev.ru/expressjs4/static-files/

**Comment by zbralex** [comment]
Стоит отметить, что мидлвары в коде вызываются в самом конце, после перечисления всех наших методов, в данном случае перед вызовом метода app.listen()
